### DISCUSSION TITLE
分享RIME上的雾凇拼音

### DISCUSSION THREAD

**Discussion by kalxd** [just-talks/tech-talks#46] [3.分享创造]
项目地址：[https://github.com/iDvel/rime-ice](https://github.com/iDvel/rime-ice)

我不用拼音，分不出好坏，从介绍上看来还不错。

**Comment by adoyle-h** [comment]
这么巧，我今天刚好给 RIME 作者赞助了一点小钱。

**Comment by kalxd** [comment]
https://emacs-china.org/t/emacs-rime/24125
转给需要的人。:D

**Comment by adoyle-h** [comment]
@kalxd 
我想在安卓的 [trime 输入法](https://github.com/osfans/trime) 安装雾凇拼音，但是就装不上。无法选择配置。你是怎么在安卓手机上使用 RIME 的，分享下配置？

**Reply by kalxd** [reply]
真是不巧，我不用rime，哪个平台都不用。
